RecoVertex/V0Producer/src/V0Fitter.cc

```diff
@@ -206,7 +203,8 @@ void V0Fitter::fitAll(const edm::Event& iEvent, const edm::EventSetup& iSetup,
       GlobalPoint vtxPos(theVtx.x(), theVtx.y(), theVtx.z());
 
       // 2D decay significance
-      const SMatrixSym3D totalCov = theBeamSpot->rotatedCovariance3D() + theVtx.covariance();
+      SMatrixSym3D totalCov = theBeamSpot->rotatedCovariance3D() + theVtx.covariance();
+      if (useVertex_) totalCov = referenceVtx.covariance() + theVtx.covariance();
```

Comment:
i added the 3D/vertex features to 76X, 75x, 74X and for the upcoming heavy
ion run
https://github.com/cms-sw/cmssw/pull/11660

but there was a bug, so this last round of PRs is fixing that bug in the
appropriate releases

On Wed, Oct 28, 2015 at 9:14 AM, Vincenzo Innocente <
notifications@github.com> wrote:

> In RecoVertex/V0Producer/src/V0Fitter.cc
> <https://github.com/cms-sw/cmssw/pull/12142#discussion_r43268712>:
>
> > @@ -206,7 +203,8 @@ void V0Fitter::fitAll(const edm::Event& iEvent, const edm::EventSetup& iSetup,
> >        GlobalPoint vtxPos(theVtx.x(), theVtx.y(), theVtx.z());
> >
> >        // 2D decay significance
> > -      const SMatrixSym3D totalCov = theBeamSpot->rotatedCovariance3D() + theVtx.covariance();
> > +      SMatrixSym3D totalCov = theBeamSpot->rotatedCovariance3D() + theVtx.covariance();
> > +      if (useVertex_) totalCov = referenceVtx.covariance() + theVtx.covariance();
>
> som why fixing 76?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/12142/files#r43268712>.
>